cpp/Makefile

```diff
@@ -14,10 +15,10 @@ install-deps:
 
```

Comment:
would be good to take care of TODO at line 12. 
maybe have OS specific deps mentioned somewhere or provide a make command for the same or is that not needed as we menetion deps already in nwaku README?